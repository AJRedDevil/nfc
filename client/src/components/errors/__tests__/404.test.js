// npm packages
import React from 'react';
import renderer from 'react-test-renderer';

// our packages
import NotFound from '../404';

test('# Not Found', () => {
  const wrapper = renderer.create(<NotFound />).toJSON();
  expect(wrapper).toMatchSnapshot();
});
