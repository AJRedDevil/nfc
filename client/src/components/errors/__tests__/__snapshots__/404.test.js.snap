// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`# Not Found 1`] = `
<div>
  <h1>
    404 Not Found
  </h1>
</div>
`;
